contracts/back_to_the_future/Cargo.toml

```diff
@@ -18,7 +18,6 @@ scale = { package = "parity-scale-codec", version = "3", default-features = fals
 scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
 
 button = { path = "../button", default-features = false }
-game_token = { path = "../game_token", default-features = false, features = ["ink-as-dependency"] }
 access_control = { path = "../access_control", default-features = false, features = ["ink-as-dependency"] }
```

Comment:
Is there a reason why this and `ink_lang` are not listed under `std`?

Reply:
Nevermind `access_control`, `ink_lang` is what I am curious about.

Reply:
only unit tests are compiled with std - no tests, no need. You could probably even remove the whole std section, that is unless metadata needs it, which I haven't checked

Reply:
ok, metadata (the part that generates ABI artifacts) is std - so you would need metadata, scale and the button and the last one only because there is an error variant that is std (also connected to test environment)